content/en/platform/corda/4.11/enterprise/network/available-network-parameters.md

```diff
@@ -11,46 +11,111 @@ title: Available network parameters
 
-* **epoch**:
-Version number of the network parameters. Starting from 1, this will always increment whenever any of the
-parameters change.
+The parameter is optional in both the network parameters and the node configuration. However, if no values are set then it needs to be specified in the flow.
+
+For example:
 
-* **whitelistedContractImplementations**:
-List of whitelisted versions of contract code.
-For each contract class there is a list of SHA-256 hashes of the approved CorDapp jar versions containing that contract.
-Read more about *Zone constraints* here api-contract-constraints
+```
+recoveryMaximumBackupInterval = 5d
+```
+or
+```
+recoveryMaximumBackupInterval = 30d
+```
 
-* **eventHorizon**:
-Time after which nodes are considered to be unresponsive and removed from network map. Nodes republish their
-`NodeInfo` on a regular interval. Network map treats that as a heartbeat from the node.
+## `packageOwnership`
 
-* **packageOwnership**:
-List of the network-wide java packages that were successfully claimed by their owners.
+The list of the network-wide Java packages that were successfully claimed by their owners.
 Any CorDapp JAR that offers contracts and states in any of these packages must be signed by the owner.
-This ensures that when a node encounters an owned contract it can uniquely identify it and knows that all other nodes can do the same.
-Encountering an owned contract in a JAR that is not signed by the rightful owner is most likely a sign of malicious behaviour, and should be reported.
+This ensures that when a node encounters an owned contract, it can uniquely identify it and knows that all other nodes can do the same.
+Encountering an owned contract in a JAR that is not signed by the rightful owner is most likely a sign of malicious behavior, and should be reported.
 The transaction verification logic will throw an exception when this happens.
 Read more about package ownership in the [Package namespace ownership]({{< relref "../node/deploy/env-dev.md#package-namespace-ownership" >}}) section.
+
+## `transactionRecoveryPeriod`
+
+When performing [Ledger Recovery](ledger-recovery.md), this value specifies how far back in days transactions are recovered. The assumption is that any transactions prior to this time have already been backed up.
+
+For example:
+
+```
+tranasctionRecoveryPeriod = 10m
+```
+
+## `whitelistedContractImplementations`
+
+The list of whitelisted versions of contract code.
+For each contract class there is a list of SHA-256 hashes of the approved CorDapp JAR versions containing that contract.
+For more information about zone constraints, see [Contract constraints]({{< relref "../cordapps/api-contract-constraints.md" >}}).

```

Comment:
```suggestion
For more information about zone constraints, see [Contract Constraints]({{< relref "../cordapps/api-contract-constraints.md" >}}).

```
Both words hyphenated in the title: https://docs.r3.com/en/platform/corda/4.10/enterprise/cordapps/api-contract-constraints.html